__tests__/toolchains.test.ts

```diff
@@ -0,0 +1,249 @@
+import io = require('@actions/io');
+import fs = require('fs');
+import path = require('path');
```

Comment:
Could you change this to the `import * as ...` syntax?  That avoids some subtle gotchas

Reply:
Doing that will throw "Cannot redefine property" errors because of the `spyOSHomedir = jest.spyOn(os, 'homedir')` in the test setup. Interestingly it works for `core` (where we do the same thing).
I am not well versed enough in either TypeScript or Jest to propose a better alternative. Any suggestions?

Reply:
I went ahead and fixed this for all except `os` for now.